.storybook/themes/news.scss

```diff
@@ -1626,7 +1648,24 @@
 							align-self: center,
 							font-size: var(--global-font-size-15),
 							justify-content: center,
-							line-height: 1.2,
+							line-height: var(--global-spacing-10),
```

Comment:
updating the xl manual promo styles? wait, is this intentional?

Reply:
Intentional... the styles are basically the same and it was a change in approach since that was merged in.  Mainly for consistency and to help with this upcoming ticket to rework line-heights.

Reply:
ah